src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -17,7 +17,7 @@ public void testCountIn() {
 
     @Test
     public void testCallCheck() {
-        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+        final DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
```

Comment:
You don't need to write final here because it is not a class attribute, it's just a variable inside a function.